torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2341,6 +2341,18 @@ RegisterOperators reg2({
           return 0;
         },
         aliasAnalysisFromSchema()),
+    Operator(
+        "aten::list(Tensor t) -> Tensor[]",
```

Comment:
The schema here is missing alias annotations, since the resulting list is a list of views over `t` they should be in the same alias set

More details: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/docs/OVERVIEW.md#aliasing-and-mutation-annotations-in-functionschema